universal-application-tool-0.0.1/app/auth/AdfsProfileAdapter.java

```diff
@@ -11,11 +11,11 @@
 import repository.UserRepository;
 
 /**
- * This class takes an existing UAT profile and augments it with the information from an AD profile.
- * Right now this is only extracting the email address, since that is all that AD provides right
- * now.
+ * This class takes an existing CiviForm profile and augments it with the information from an AD
+ * profile. Right now this is only extracting the email address, since that is all that AD provides
+ * right now.
  */
-public class AdfsProfileAdapter extends UatProfileAdapter {
+public class AdfsProfileAdapter extends CiviFormProfileAdapter {
```

Comment:
Question (blocking): given this profile adapter is specific to ADFS shouldn't it continue to include that in its name?